packages/server/src/user/user.service.ts

```diff
@@ -86,12 +87,20 @@ export class UserService {
    * @returns `User` object, or throw `NotFoundError` when not exist
```

Comment:
Should update the comment to reflect the change in how a not found is handled

Reply:
Thanks for that